app/components/typography/NumberBadge.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+interface NumberBadgeProps {
+  count: number;
+}
+
+const NumberBadge: React.FC<NumberBadgeProps> = ({ count }) => {
+  return (
+    <span className="font-medium text-xs leading-5 text-leafGreen">
+      {count} person Enrolled
```

Comment:
The hardcoded text "person Enrolled" does not handle pluralization correctly when count is greater than 1. This can be improved by dynamically adjusting the text based on the count.
```suggestion
      {count} {count === 1 ? "person Enrolled" : "people Enrolled"}
```

Reply:
done